module/VuFind/src/VuFind/Db/Service/UserService.php

```diff
@@ -228,7 +254,7 @@ public function updateUserEmail(
      */
     public function addUserDataToSession(UserEntityInterface $user): void
     {
-        if ($user instanceof UserRow) {
+        if ($user) {
```

Comment:
I think you can completely get rid of the if/else here, since `$user` will never be false.